grakn-graql-shell/src/main/java/ai/grakn/graql/GraqlShell.java

```diff
@@ -169,6 +169,8 @@ public static void runShell(String[] args, String version, String historyFilenam
         options.addOption("f", "file", true, "graql file path to execute");
         options.addOption("r", "uri", true, "uri to factory to engine");
         options.addOption("b", "batch", true, "graql file path to batch load");
+        options.addOption("s", "size", true, "the size of the batches");
+        options.addOption("a", "active", true, "the number of active tasks");
```

Comment:
It would probably good to include in the help text that this is only for batch loads.